src/pages/settings/Wallet/Card/BaseGetPhysicalCard.tsx

```diff
@@ -109,24 +109,25 @@ function BaseGetPhysicalCard({
     const styles = useThemeStyles();
     const isRouteSet = useRef(false);
 
+    const domainCards = CardUtils.getDomainCards(cardList)[domain] || [];
+    const cardToBeIssued = domainCards.find((card) => !card?.nameValuePairs?.isVirtual && card?.state === CONST.EXPENSIFY_CARD.STATE.STATE_NOT_ISSUED);
+    const cardID = cardToBeIssued?.cardID ?? 0;
```

Comment:
I think it would be good to use `cardID` as string (like `cardToBeIssued?.cardID?.toString() ?? ‘’`) instead of calling `toString` many times. We need `cardID` as a number only in `requestPhysicalExpensifyCard` where we can use `cardToBeIssued?.cardID ?? 0`. This will make the code look neater too. What do you think?